managed-databases/document-database/api-cli/backup-and-restore.mdx

```diff
@@ -0,0 +1,109 @@
+---
+meta:
+  title: Back up and restore Document Databases with MongoDB® tools
+  description: This page shows you how to back up and restore Document Databases with MongoDB® command line tools
+content:
+  h1: Back up and restore Document Databases with MongoDB® tools
+  paragraph: This page shows you how to back up and restore Document Databases with MongoDB® command line tools
+tags: databases mongodb ferretdb document backup database-nodes
+categories:
+  - managed-databases
+  - postgresql-and-mysql
+---
+
+Managed Document Databases provide fully-managed document Database Instances, with a MongoDB®-compatible API layer over a PostgreSQL engine to store and retrieve data.
+
+This means you can use MongoDB® command line tools to use and manage your Document Databases.
+
+This tutorial shows how to backup and restore your Document Databases with the MongoDB® `mongodump` and `mongorestore` command line database tools.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+  - You have a [MongoDB®-compatible client](https://www.mongodb.com/try/download/shell) installed
+  - You have [created a Document Database Instance](/managed-databases/document-database/how-to/create-a-database-instance)
+</Message>
```

Comment:
In my experience, having mongo shell installed (ie. a mongo client) doesn't include mongodump and mongodb restore tools. 
```suggestion
  - You have [mongodump and mongorestore tools](https://www.mongodb.com/docs/database-tools/installation/installation/) installed
</Message>
```